requirements-test.txt

```diff
@@ -1,27 +1,28 @@
 boto3==1.16.63
 botocore==1.19.63
-certifi==2020.11.8
+certifi==2020.12.5
 click==8.0.0a1
-fsspec==0.8.4
+fsspec==0.8.5
 jmespath==0.10.0
-marshmallow==3.9.1
+marshmallow==3.10.0
 matplotlib==3.3.3
-numpy==1.19.4
-pandas==1.1.4
+numpy>=1.19.4,<=1.20.0
+pandas>=1.1.4
```

Comment:
The requirements.txt files are meant to lock dependencies (and thus the `==` equality). This is to ensure that the runtime environment is identical across different machines.

For expressing minimal requirements, that should be done in `setup.cfg` FYI

Reply:
```suggestion
pandas==1.1.4
```